.buildkite/pipeline.yml

```
@@ -1,13 +1,13 @@
 env:
-  BASE_CONTAINER: 207654339136.dkr.ecr.us-east-1.amazonaws.com/mountainroseherbs/spree-test
-  TEST_CONTAINER: 207654339136.dkr.ecr.us-east-1.amazonaws.com/mountainroseherbs/spree-test:$BUILDKITE_COMMIT
+  BASE_CONTAINER: 207654339136.dkr.ecr.us-east-1.amazonaws.com/mountainroseherbs/spree-test:$PREVIOUS_SHA
+  TEST_CONTAINER: 207654339136.dkr.ecr.us-east-1.amazonaws.com/mountainroseherbs/spree-test:$COMMIT_SHA
 
 steps:
   - name: Test Container
     command: |
-      set -e
-      eval "$(aws ecr get-login)"
-      docker pull "$BASE_CONTAINER"
+      source strict-mode.sh
+
+      docker-pull-ecr "$BASE_CONTAINER" || true
```

Do we want this build step always to be succesfull, even when `docker-pull-ecr` fails? I think we should propagate the failure instead to shadow it, as it shows an infrastructure problem we should be aware about.